drizzle-kit/src/serializer/pgSchema.ts

```diff
@@ -555,10 +554,7 @@ export const PgSquasher = {
 		return `${idx.name};${
 			idx.columns
 				.map(
-					(c) =>
-						`${c.expression}--${c.isExpression}--${c.asc}--${c.nulls}--${
-							c.opclass && vectorOps.includes(c.opclass) ? c.opclass : ''
-						}`,
+					(c) => `${c.expression}--${c.isExpression}--${c.asc}--${c.nulls}--${c.opclass ? c.opclass : ''}`,
```

Comment:
why is this change needed?

Reply:
This was changed to address https://github.com/drizzle-team/drizzle-orm/issues/2935. In the ORM code, `PgIndexOpClass` includes more than just vector operations, so I assume they should be supported in Kit as well. The removed check only allowed vector ops to be generated.